lib/storage/providers/redis.js

```diff
@@ -204,6 +204,51 @@ StorageRedis.prototype.getServiceOutagesSince = function (service, timestamp, ca
 };
 
 /**
+ * Get outage history for all services
+ * @param services
+ * @param timestamp
+ * @param maxItems
```

Comment:
right now it's really a `maxItems` as we end up returning `servicesOutages.slice(0, maxItems)`.
The right question would be, is there a more efficient way without breaking the current redis data structure to get only 10 instead total. So far I get (at most) 10 of each in order to ensure I get enough of the wanted result in that final slice.

We could also think of a maxItemsPerService but my problem with it is that it totally doesn't say anything on the result size (if we have 1 service we get 0-10, if we have 10000 services we get 0-10000) which is less predictable than a flat 0-10.